bdd/features/tipo8/_refund.feature

```diff
@@ -0,0 +1,18 @@
+Feature: A merchant gets refunded if a transaction is discounted
+
+Background:
+    Given the initiative is "Scontoditipo8"
+    And the citizen A is onboard
```

Comment:
Here should be added: 
- And the merchant 1 is qualified
- And the citizen A is 25 years old at most